tests/unit/v1/test_document.py

```diff
@@ -62,30 +62,31 @@ def test_constructor_invalid_kwarg(self):
 
     @staticmethod
     def _make_commit_repsonse(write_results=None):
-        from google.cloud.firestore_v1.proto import firestore_pb2
+        from google.cloud.firestore_v1.types import firestore
 
-        response = mock.create_autospec(firestore_pb2.CommitResponse)
+        response = mock.create_autospec(firestore.CommitResponse)
         response.write_results = write_results or [mock.sentinel.write_result]
         response.commit_time = mock.sentinel.commit_time
         return response
 
     @staticmethod
     def _write_pb_for_create(document_path, document_data):
-        from google.cloud.firestore_v1.proto import common_pb2
-        from google.cloud.firestore_v1.proto import document_pb2
-        from google.cloud.firestore_v1.proto import write_pb2
+        from google.cloud.firestore_v1.types import common
+        from google.cloud.firestore_v1.types import document
+        from google.cloud.firestore_v1.types import write
         from google.cloud.firestore_v1 import _helpers
 
-        return write_pb2.Write(
-            update=document_pb2.Document(
+        return write.Write(
+            update=document.Document(
                 name=document_path, fields=_helpers.encode_dict(document_data)
             ),
-            current_document=common_pb2.Precondition(exists=False),
+            current_document=common.Precondition(exists=False),
         )
 
     def test_create(self):
         # Create a minimal fake GAPIC with a dummy response.
-        firestore_api = mock.Mock(spec=["commit"])
+        firestore_api = mock.Mock()
+        firestore_api.commit.mock_add_spec(spec=["commit"])
```

Comment:
From what I can tell these are functionality equivalent. Curious what the reason for this change is.

Reply:
the mock library isn't a fan of nesting...